store/storetest/command_webhook_store.go

```diff
@@ -49,25 +43,22 @@ func testCommandWebhookStore(t *testing.T, ss store.Store) {
 	h2, err = cws.Save(h2)
 	require.Nil(t, err)
 
-	if _, err := cws.Get(h2.Id); err == nil || err.StatusCode != http.StatusNotFound {
-		t.Fatal("Should have set the status as not found for expired webhook")
-	}
+	_, err = cws.Get(h2.Id)
+	assert.NotNil(t, err, "Should have set the status as not found for expired webhook")
```

Comment:
I think all those `(not)nil` assertions to `err` should use `require` but will defer to other reviewers.

```suggestion
	require.NotNil(t, err, "Should have set the status as not found for expired webhook")
```

Reply:
Let me know and I can change to require.

Reply:
I prefer `require` myself, but I'm not opposed to `assert`.

Reply:
Lets do `require` then.

Reply:
Done.